### PR TITLE
Update list of stimuli

### PR DESCRIPTION
**Pull request by jthermiz**
I updated the list of stimuli per #44 . I tried to write a test case for `wav_manager.py` but it's failing bc I don't understand what the arguments should be. @jihyunbak What should the args for `WavManager` be? 

Also I fixed the accidental changes I made to `tdt_manager.py`

### CODE REVIEW

**Comment by jthermiz** [comment]
Sorry for the excessive number of commits. I was having trouble syncing my forked repo with BouchardLab/nsds_lab_to_nwb

**Comment by jihyunbak** [comment]
Thanks for this @jthermiz ! The second argument for WavManager is the stimulus metadata, which is a dictionary object. For the purpose of testing `WavManager.get_stim_file`, maybe try passing an empty dict `{}` — to test the other methods, I think it would be easiest to create an `NWBBuilder` object and access `builder.metadata['stimulus']`

**Comment by jihyunbak** [comment]
@jthermiz I ended up adding some commits myself, basically adding new `test_wav_manager` - I hope you don't mind my pushing to your main branch...

File: nsds_lab_to_nwb/components/tdt/tdt_manager.py [hunk 1]
```diff
@@ -30,9 +30,9 @@ def extract(self, device_name, dev_conf, electrode_table_region):
             result = self.tdt_reader.get_data(device_name)
         else:
             logger.info('- stream {} not found. Available stream list: {}'.format(device_name, stream_list))
-            results = None
+            result = None
             # try alternative (device, stream) name pairs
-            alternative_device_names = [('ECoG', 'Wave')]
```
Comment (jihyunbak) [inline nsds_lab_to_nwb/components/tdt/tdt_manager.py:35]: Does this mean 'Wave' is actually equivalent to 'Poly', not 'ECoG'?

Comment (jihyunbak) [reply nsds_lab_to_nwb/components/tdt/tdt_manager.py:35]: This pair `('ECoG', 'Wave')` means "the device named 'ECoG' may go with a stream called 'Wave' so let's look for that alternative name" — so I'm pretty sure we can revert this back!

File: nsds_lab_to_nwb/_data/list_of_stimuli.yaml [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
+#               that stores parameters that describe auditory stimlus
 # ---------------------------------------------------------------
 
 tone150:
     alt_names: ['NC Tone 150']
     description: ''
-    file: 'Tone150/freq_resp_area_stimulus_signal_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    audio_path: 'Tone150/freq_resp_area_stimulus_signal_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    marker_path: 'Tone150/freq_resp_area_stimulus_trigger_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    parameter_path: 'Tone150/Tone100_stim.stimVls.mat'
```
Comment (JesseLivezey) [inline nsds_lab_to_nwb/_data/list_of_stimuli.yaml:20]: Can we rename this .mat file and then update this field? ^This^ is the current filename, but I think it was a mistake whant the .mat was created

Comment (jthermiz) [reply nsds_lab_to_nwb/_data/list_of_stimuli.yaml:20]: I renamed the .mat as the same sname at the audio_path file name

**Comment by jthermiz** [comment]
What's up with the __init__.py in _data? Should I just ignore this?

File: nsds_lab_to_nwb/components/tdt/tdt_manager.py (hunk 1 above)
Comment (jthermiz) [reply nsds_lab_to_nwb/components/tdt/tdt_manager.py:35]: Ok my bad. Yes we should revert

File: tests/test_wav_manager.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import os
```
Comment (jthermiz) [inline tests/test_wav_manager.py:1]: I run into this error when I try to run this:

FileNotFoundError: [Errno 2] No such file or directory: '../../_data/list_of_stimuli.yaml'

I think you implemented the importlib.resource thing which is how you get it to work?

File: nsds_lab_to_nwb/_data/list_of_stimuli.yaml (hunk 2 above)
Comment (jihyunbak) [reply nsds_lab_to_nwb/_data/list_of_stimuli.yaml]: Thanks for this @jthermiz ! Quick question - what's the path to the stimulus library directory that you are working with?

File: tests/test_wav_manager.py (hunk 3 above)
Comment (jihyunbak) [reply tests/test_wav_manager.py:1]: Sorry for the error, I think the importlib.resources tool should get this to work - was this before this change was applied?

File: nsds_lab_to_nwb/_data/list_of_stimuli.yaml (hunk 2 above)
Comment (jihyunbak) [reply nsds_lab_to_nwb/_data/list_of_stimuli.yaml]: Moved discussion to #36

**Pull request merged**